src/routes/write-exercise/components/PopoverContent.tsx

```diff
@@ -1,35 +1,36 @@
-import React, { ReactElement } from 'react'
-import { View, Text, StyleSheet } from 'react-native'
+import React from 'react'
 import { InfoIcon } from '../../../../assets/images'
 import { widthPercentageToDP as wp } from 'react-native-responsive-screen'
 import { COLORS } from '../../../constants/colors'
 import labels from '../../../constants/labels.json'
+import styled from 'styled-components/native'
 
-export const styles = StyleSheet.create({
-  container: {
-    flexDirection: 'row',
-    alignItems: 'center',
-    backgroundColor: COLORS.lunesBlack,
-    width: wp('80%'),
-    height: 60,
-    paddingVertical: 8,
-    paddingHorizontal: 9,
-    borderRadius: 2
-  },
-  message: {
-    color: COLORS.lunesWhite,
-    fontSize: wp('3.5%'),
-    fontWeight: 'normal',
-    fontFamily: 'SourceSansPro-Regular',
-    width: wp('60%'),
-    marginLeft: 8
-  }
-})
-const PopoverContent = (): ReactElement => (
-  <View style={styles.container}>
+const StyledComponent = styled.View`
```

Comment:
Outer components can always be named as `Container`. In this case you can name it `StyledPopoverContentContainer` or just `StyledContainer`